core/commonui/lemmyui/src/commonMain/kotlin/com/github/diegoberaldin/raccoonforlemmy/core/commonui/lemmyui/CommentCard.kt

```diff
@@ -176,14 +179,24 @@ fun CommentCard(
                             PostCardBody(
                                 text = comment.text.orEmpty(),
                                 autoLoadImages = autoLoadImages,
-                                onClick = onClick,
+                                onClick = {
+                                    if (textSelection.value) {
+                                        focusManager.clearFocus(true)
+                                        textSelection.value = false
```

Comment:
if the previous suggestion is applied, this can just become as easy as:

```suggestion
                                        textSelection = false
```